docs/examples/transforms/TransformsEval.ipynb

```diff
@@ -0,0 +1,1622 @@
+{
```

Comment:
<p>any idea why summary is worse than default</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9005/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
it's possible there's a bug

Reply:
investigating right now

Reply:
actually did a quick check, seems fine. my hypothesis: 
- i wouldn't overfit to this dataset, it's quite toy
- extracting summary and putting in text can bias embeddings for retrieval or confuse llm with more context